src/utils/permalinks/Permalinks.ts

```diff
@@ -331,8 +331,8 @@ export function tryTransformEntityToPermalink(entity: string): string {
     if (entity[0] === '@') return makeUserPermalink(entity);
     if (entity[0] === '+') return makeGroupPermalink(entity);
 
-    // Then try and merge it into a permalink
-    return tryTransformPermalinkToLocalHref(entity);
+    // does not need conversion
+    return entity;
```

Comment:
The documentation and change don't appear to match - it should return null if the entity is unparsable, but here we now assume that it's always valid.

Can we update the implementation and/or the docs please?